### PR TITLE
Recent New Sources Counter

### PR DESCRIPTION
**Pull request by profjsb**
Adds a source info widget displaying the number of sources created in the last X days (with X configurable) by the end user.

image

Also, starts the page reformatting process, in anticipation of new widgets.

### CODE REVIEW

File: package.json
Comment (stefanv) [inline package.json:51]: Worth a dependency addition?

File: skyportal/handlers/api/internal/source_counts.py
Comment (stefanv) [inline skyportal/handlers/api/internal/source_counts.py:7]: Necessary to have this as a preference? Can we just hard code to a week for simplicity? Doesn't seem like this is a very important scientific factoid.

File: static/js/components/HomePage.jsx
Comment (stefanv) [inline static/js/components/HomePage.jsx:22]: Can we leave this for K to address later? Bound to cause conflicts, and not sure we want to hold up this PR for any tuning here.

Comment (kmshin1397) [reply static/js/components/HomePage.jsx:22]: Stefan has a good point - we'll be changing these a lot as all the new widgets we have planned for this iteration come in. Might be better to just do a separate PR at the end to reorganize all the widgets at once after they get in. We can maybe just change these locally to test the new widgets, but not include any layout changes in the PRs until we have a more stable set of home page widgets in.

File: skyportal/handlers/api/internal/source_counts.py
Comment (profjsb) [reply skyportal/handlers/api/internal/source_counts.py:7]: I think it's fine to leave it in. Some people may want to see just in the last day, others the last week.

File: static/js/components/HomePage.jsx
Comment (profjsb) [reply static/js/components/HomePage.jsx:22]: Good idea. Let's leave in these changes and then in future PRs hold off

File: package.json
Comment (profjsb) [reply package.json:51]: It's pretty small (24kb) and adds some motion to the dashboard.

File: static/js/components/SourceCounts.jsx
Comment (stefanv) [inline static/js/components/SourceCounts.jsx:34]: We use non-inline styles now (please see the style guide for how to add style classes).

**Comment by stefanv** [comment]
Also missing a test.

**Comment by kmshin1397** [comment]
@profjsb I had some of the same test errors as you on my widget PR and resolved them with these commits:

https://github.com/skyportal/skyportal/pull/736/commits/5f03b2b48fe1cf34b1c043113d28311d7a8a7f68
https://github.com/skyportal/skyportal/pull/736/commits/ef206c47120c78d3a100bfeb72edd8cd47376b0f

Also, the merge conflicts should be from my widget drag handle PR that was merged this morning - I'll propose a patch to this PR to resolve those for you.

**Comment by profjsb** [comment]
> @profjsb I had some of the same test errors as you on my widget PR and resolved them with these commits:
> 
> [5f03b2b](https://github.com/skyportal/skyportal/commit/5f03b2b48fe1cf34b1c043113d28311d7a8a7f68)
> [ef206c4](https://github.com/skyportal/skyportal/commit/ef206c47120c78d3a100bfeb72edd8cd47376b0f)
> 
> Also, the merge conflicts should be from my widget drag handle PR that was merged this morning - I'll propose a patch to this PR to resolve those for you.

@kmshin1397 ok: would you mind pushing your changes directly to this PR?

**Review by kmshin1397 — APPROVED** [review]

**Review by kmshin1397 — APPROVED** [review]

**Comment by kmshin1397** [comment]
@stefanv Can we merge this now (I don't have write access)? GA error is with ppa dependency and unrelated

**Comment by stefanv** [comment]
Thanks @profjsb @kmshin1397!

**Pull request merged**
